test/onnx/test_fx_to_onnx.py

```diff
@@ -351,34 +350,50 @@ def forward(self, x):
             fake_model = Model()
             fake_x = torch.rand(5, 2, 2)
 
-        # TODO: Split each scenario on its own test case
-        # Scenario 1: Fake model and fake input WITHOUT fake_context
-        with self.assertRaises(torch.onnx.OnnxExporterError):
-            export_options = ExportOptions(fake_context=None)
-            _ = torch.onnx.dynamo_export(
-                fake_model, fake_x, export_options=export_options
-            )
+            # TODO: Split each scenario on its own test case
+            # Scenario 1: Fake model and fake input WITHOUT ExportOptions(fake_context=...)
+            with self.assertRaises(torch.onnx.OnnxExporterError):
+                export_options = ExportOptions(fake_context=None)
+                _ = torch.onnx.dynamo_export(
+                    fake_model, fake_x, export_options=export_options
+                )
 
-        # Scenario 2: Fake model and real input WITHOUT fake_context
-        with self.assertRaises(torch.onnx.OnnxExporterError):
-            export_options = ExportOptions(fake_context=None)
-            _ = torch.onnx.dynamo_export(
-                fake_model, real_x, export_options=export_options
-            )
+            # Scenario 2: Fake model and real input WITHOUT fake_context
+            with self.assertRaises(torch.onnx.OnnxExporterError):
+                export_options = ExportOptions(fake_context=None)
+                _ = torch.onnx.dynamo_export(
+                    fake_model, real_x, export_options=export_options
+                )
 
-        # Scenario 3: Real model and real input WITH fake_context
-        with self.assertRaises(torch.onnx.OnnxExporterError):
-            export_options = ExportOptions(fake_context=fake_context)
-            _ = torch.onnx.dynamo_export(
-                real_model, real_x, export_options=export_options
-            )
+            # Scenario 3: Real model and real input WITH fake_context
+            with self.assertRaises(torch.onnx.OnnxExporterError):
+                export_options = ExportOptions(fake_context=fake_context)
+                _ = torch.onnx.dynamo_export(
+                    real_model, real_x, export_options=export_options
+                )
 
-        # Scenario 4: Fake model and real input WITH fake_context
-        with self.assertRaises(torch.onnx.OnnxExporterError):
-            export_options = ExportOptions(fake_context=fake_context)
-            _ = torch.onnx.dynamo_export(
-                fake_model, real_x, export_options=export_options
+            # Scenario 4: Fake model and real input WITH fake_context
+            with self.assertRaises(torch.onnx.OnnxExporterError):
+                export_options = ExportOptions(fake_context=fake_context)
+                _ = torch.onnx.dynamo_export(
+                    fake_model, real_x, export_options=export_options
+                )
+
+    def test_fake_tensor_mode_huggingface_bigscience__bloom_560m(self):
```

Comment:
Do we need the same model in both runtime/no_runtime tests?

Reply:
when the one with ORT passes, the other one can be deleted.